services/market/shared/market/sources/index.js

```diff
@@ -13,9 +13,25 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { reloadPricesFromBinance, getBinancePricesFromDB } = require('./binance');
-const { reloadPricesFromBittrex, getPricesFromBittrex } = require('./bittrex');
-const { reloadPricesFromKraken, getPricesFromKraken } = require('./kraken');
+const {
+	reloadPricesFromBinance,
+	getBinancePricesFromDB,
+} = require('./binance');
+
+const {
+	reloadPricesFromBittrex,
+	getPricesFromBittrex,
+} = require('./bittrex');
+
+const {
+	reloadPricesFromExchangerateapi,
+	getExchangeratesapiPricesFromDB,
+} = require('./exchangeratesapi');
+
+const {
+	reloadPricesFromKraken,
+	getPricesFromKraken,
+} = require('./kraken');
```

Comment:
Here it could actually inherit one simple interface, with standard functions `{ reload, getFromCache }`.
The `index.js` wouldn't be necessary either.